_commands/basics/echo.md

```diff
@@ -3,22 +3,71 @@
 
 echo
 ---
-TODO: Add documentation for this command by submitting a pull request.
+
+`echo` is used to display message on screen, and write given string to standard output, with each a space between each and a newline after the last one.  `echo` is implemented as builtin command in many command shells like bash, ksh, csh.
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ echo "Hello World"
+Hello World
+$ echo Hello World
+Hello World
 ~~~
 
 <!--more-->
 
+
 ### Useful Options / Examples
 
 #### Example command
+`echo` output can be redirected into a file for easy use. 
+~~~ bash
+$ echo Hello World > example.txt
+$ cat example.txt
+Hello World
+~~~
+#### Example command
+`echo` output can also be appended to a file.
```

Comment:
This example isn't specific to the `echo` command, it deals more with the redirection operator and the append operator. I'd be fine with taking this example out.